src/plugins/plugins.h

```diff
@@ -392,11 +366,13 @@ struct vmi_lock_guard
 
     ~vmi_lock_guard()
     {
-        unlock();
+        drakvuf_release_vmi(drakvuf);
     }
 
+private:
     drakvuf_t drakvuf;
-    vmi_instance_t vmi;
+public:
+    vmi_instance_t vmi; // TODO: make field private
```

Comment:
Why doesn't it work as private as-is?

Reply:
Some code actively accesses this field as public member. I weel need time to fix this in the next step.